docs/framework-introscope_agent.md

```diff
@@ -23,6 +23,7 @@ The credential payload of the service may contain the following entries:
 
 | Name | Description
 | ---- | -----------
+|`agent_defaultProcessName`| (Optional) The name that is specified for the agent process. If not specified, default value is the application name.
```

Comment:
You're comfortable that this doesn't appear to be consistent with other names in this payload (use of camel case)?

Reply:
Yes, I am fine that because we decided that adding extra underscores means more typing for the user.

Reply:
And the inconsistency more difficult to predict.  But that's fine.